server/Controllers/GameControllers/snake.js

```diff
@@ -1,12 +1,117 @@
+const { SNAKE } = require('./gameConstants');
+
 class Snake {
-  constructor(playerId, initialPositionX, initialPositionY,
-    initialDirection, snakeColor, initialSnakeLength) {
+  /**
+   * @param  {string} playerId
+   * @param  {Object} initialProperties initial values for snake body and motion.
+   * @param  {Function} eatFood Function to check that snake's next movement will be eating food.
+   * @param  {Array} rivalBody Rival snake's body coordinates
+   * @param  {number} intervalNumber
```

Comment:
to clear the current setInterval( I just got to know, in node it's an object, so It will make more sense if I give it a name gameInterval), that has been executed on the server after a fixed time. I need to clear/stop the game when the snake has lost the game.

Reply:
`gameInterval` is way better then `intervalNumber`

Reply:
Yup will update this. thanks